.github/workflows/pr-linter.yml

```diff
@@ -98,6 +98,8 @@ jobs:
           ONEDNN_ACTION: configure
 
       - name: Build oneDNN
-        run: .github/automation/build_linters.sh
+        run: |
+            .github/automation/build_linters.sh | tee build.log
+            grep -q "warning" build.log && exit 1 || true
```

Comment:
Consider this:
```
grep -i "warning:" build.log | sort -u && exit 1 || true
```

Clang-tidy may generate duplicate warnings during build, so having a summary of unique warnings in the end is useful.

Also having a colon added to pattern would reduce false matches.